frontend/src/metabase/query_builder/reducers.js

```diff
@@ -1,7 +1,7 @@
 import { handleActions } from "redux-actions";
 import { assoc, merge } from "icepick";
 import _ from "underscore";
-import Utils from "metabase/lib/utils";
+import { copy } from "metabase/lib/utils";
```

Comment:
TBH, I don't see any problem with using namespace for Utils moreover I'd prefer using namespace for it as utils usually have pretty simple names which can collide

Reply:
What I complained about in the different PR is using namespace in the same file as it was done with compareVersions

Reply:
> utils usually have pretty simple names which can collide

This can be a good thing, e.g. to help detect duplications (e.g. [`isEmail`](https://github.com/metabase/metabase/blob/45dcf57327c8f50826686adbb3d739c1de461520/frontend/src/metabase-lib/types/utils/isa.js#L196)) or to improve naming (if functions work differently, they should probably have a different name).

----

+1 on namespace removal from my side (consistency)